connectors/atp/app.py

```diff
@@ -36,6 +37,41 @@ def setup(self):
         context().inc_importer = IncrementalImport()
 
 
+    def get_schema_extension(self):
+        return SchemaExtension(
+            key='f1532f2f-32bf-4c39-9bcb-89643fcffd05',  # generate your own UUID key!
+            owner='Reference Connector',
+            version='1',
+            schema='''
+               {
+                   "vertices": [
+                        {
+                           "name": "asset",
```

Comment:
The `os` information for asset should be in `application` collection with `is_os` `True`.